src/Resources/app/administration/src/module/sw-sales-channel-detail/extension/sw-sales-channel-detail-base/index.js

```diff
@@ -1,22 +1,7 @@
-import template from './sw-sales-channel-detail-base.html.twig';
-
-const { Component, Defaults } = Shopware;
-const { Criteria } = Shopware.Data;
-
-Component.override('sw-sales-channel-detail-base', {
-    template,
-
+Shopware.Component.override('sw-sales-channel-detail-base', {
     computed: {
         languageCriteria() {
-            return (new Criteria())
-                .addAssociation('swagLanguagePackLanguage')
-                .addAssociation('locale')
-                .addSorting(Criteria.sort('name', 'ASC'))
-                .addFilter(Criteria.multi('OR', [
-                    Criteria.equals('extensions.swagLanguagePackLanguage.id', null),
-                    Criteria.equals('extensions.swagLanguagePackLanguage.salesChannelActive', true),
```

Comment:
This active flag can still be used in the plugin settings. But technically, disabling a Language via LanguagePack settings won't disable the `language.active` flag. For transitioning away from the LanguagePack and towards the new Language system, I'd ignore this field.

Reply:
Good point.

Reply:
Reverted it for now. But I don't think we need to deprecate it, since at that point we can drop the LanguagePack anyway